doc/playbook.rst

```diff
@@ -470,17 +470,17 @@ To restore the APDB to a clean state, run the following:
 .. code-block:: sh
 
    psql -h usdf-prompt-processing-dev.slac.stanford.edu lsst-devl rubin -c 'drop schema "pp_apdb_latiss" cascade;'
-   make_apdb.py -c namespace="pp_apdb_latiss" \
-       -c db_url="postgresql://rubin@usdf-prompt-processing-dev.slac.stanford.edu/lsst-devl"
+   apdb-cli create-sql --namespace="pp_apdb_latiss" \
+       "postgresql://rubin@usdf-prompt-processing-dev.slac.stanford.edu/lsst-devl" apdb_config_latiss.py
```

Comment:
Replace the original `psql` command with the `--drop` flag?

Reply:
There is a subtle difference - `--drop` only removes tables that it thinks should exist (based on current configuration). IF configuration changed beween two `create-sql` executions, then some tables may not be removed. `DROP SCHEMA` is cleaner in that respect.

Reply:
Yes, I did notice that, but I'm assuming new versions are more likely to add tables than remove them. 😀 Calling "drop schema" was mostly because we needed a  way to ensure compatibility "by hand".